roles/edpm_bootstrap/defaults/main.yml

```diff
@@ -33,6 +33,8 @@ edpm_bootstrap_packages_bootstrap:
   - nftables
   - NetworkManager
   - openstack-selinux
+  - openvswitch
+  - os-net-config
```

Comment:
already installed by edpm_network_config, which actually includes edpm_bootstrap/tasks/packages.yml, and then the very next task is to install os-net-config, so this could be redundant.

Reply:
but os-net-config mantioned in meta default arguments_spec.yaml,
can i remove this os-net-config also in arguments_spec default?